changelog/3822.feature.rst

```diff
@@ -0,0 +1 @@
+Add support for parfive 1.1. This sets a limit on the number of open connections to JSOC when downloading files to 10.
```

Comment:
I thought it was 8?

Reply:
We always set it to 8 but we allow 10 if explicitly passed by the user.

Reply:
Oh i see.